pkg/api/azenvtypes.go

```diff
@@ -244,14 +246,15 @@ var (
 			ResourceManagerVMDNSSuffix: "cloudapp.chinacloudapi.cn",
 		},
 		OSImageConfig: map[Distro]AzureOSImageConfig{
-			Ubuntu: SovereignCloudsUbuntuImageConfig,
-			RHEL:   DefaultRHELOSImageConfig,
-			CoreOS: DefaultCoreOSImageConfig,
-			AKS:    SovereignCloudsUbuntuImageConfig,
+			Ubuntu:          SovereignCloudsUbuntuImageConfig,
+			RHEL:            DefaultRHELOSImageConfig,
+			CoreOS:          DefaultCoreOSImageConfig,
+			AKS:             SovereignCloudsUbuntuImageConfig,
+			AKSDockerEngine: SovereignCloudsUbuntuImageConfig,
 		},
 	}
 
-	// AzureCloudSpecEnvMap is the environment configuration map for all the Azure cloid environments.
+	// AzureCloudSpecEnvMap is the environment configuration map for all the Azure cloud environments
```

Comment:
Super nit-pick--can we leave the period intact? Go comments are meant to be complete sentences.

Reply:
whoops for sure

Reply:
added back